torch/nn/parallel/distributed.py

```diff
@@ -541,6 +541,9 @@ class DistributedDataParallel(Module, Joinable):
         broadcast_buffers (bool): Flag that enables syncing (broadcasting)
                           buffers of the module at beginning of the ``forward``
                           function. (default: ``True``)
+        init_sync (bool): Whether to sync during initialization to verify param
+                          shapes and broadcast parameters and buffers.
```

Comment:
maybe also more explicitly call out that if false, the user is expected to ensure each replica has the same parameters before starting to use DDP

Reply:
Good call out, updated!